apps/scores_plot.py

```diff
@@ -0,0 +1,48 @@
+from inpainting.dataset import Data, ResizeTransform, NoiseSampler, ConditionSampler
+from gan.trainer import GanTrainer
+import torch
+print(torch.__version__)
+from torch.utils.data import DataLoader, random_split
+
+from inpainting.visualizer import Visualizer
+from performance.estimator import FIDEstimator
+from argparse import ArgumentParser
+import importlib
+
+if __name__ == '__main__':
+    parser = ArgumentParser()
+    parser.add_argument('--conf', default='inpainting.celeba_config')
+    args = parser.parse_args()
+    conf = importlib.import_module(args.conf)
+    # torch.cuda.set_device(conf.CUDA_DEVICE)
+    device = torch.device(conf.DEVICE)
+
+    transform = ResizeTransform()
+    data = Data(conf.DATA_PATH, transform)
+    train_size = int(0.8 * len(data))
+    valid_size = len(data) - train_size
+    train_data, valid_data = torch.utils.data.random_split(data, [train_size, valid_size])
+    train_loader = DataLoader(train_data, batch_size=conf.BATCH_SIZE, num_workers=conf.NUM_WORKERS, shuffle=True)
+    valid_loader = DataLoader(valid_data, batch_size=conf.BATCH_SIZE, num_workers=conf.NUM_WORKERS, shuffle=True)
+    print('Dataset size: ', len(data))
+    if hasattr(conf, 'conditions'):
+        from gan.conditional_gan import Generator5Net, Discriminator5
```

Comment:
don't really understand all this imports in the code

Reply:
this script can work with both conditional and unconditional gan. But they are defined in different modules